service-tests/helpers/validators.js

```diff
@@ -11,6 +11,8 @@ const isVPlusTripleDottedVersion = withRegex(/^v[0-9]+.[0-9]+.[0-9]+$/);
 
 const isVPlusDottedVersionAtLeastOne = withRegex(/^v\d+(\.\d+)?(\.\d+)?$/);
 
+const isCommaSeperatedPythonVersions = withRegex(/^([0-9]+.[0-9]+[,]?[ ]?)+$/);
```

Comment:
Could you move this to `service-tests/pypi.js`?

This has come up before actually, so perhaps you could add a comment here for the next developer?

> Regexes here are used by more than one service. Regexes used by a single service should be added to the service's test file.